packages/prosess-vedtak/src/components/brev/BrevPanel.jsx

```diff
@@ -99,13 +106,15 @@ export const BrevPanel = props => {
 
   const harAutomatiskVedtaksbrev = kanHaAutomatiskVedtaksbrev(tilgjengeligeVedtaksbrev);
   const harFritekstbrev = kanHaFritekstbrev(tilgjengeligeVedtaksbrev);
+  const harAlternativeMottakere = kanOverstyreMottakere(tilgjengeligeVedtaksbrev);
 
   const fritekstbrev = harFritekstbrev && (
     <>
       <FritekstBrevPanel
         readOnly={readOnly}
         sprakkode={sprakkode}
         previewBrev={automatiskBrevUtenValideringCallback}
+        tilgjengeligeVedtaksbrev={tilgjengeligeVedtaksbrev}
         harAutomatiskVedtaksbrev={harAutomatiskVedtaksbrev}
```

Comment:
Fritekstpanel mangler intl, tilgjengeligeVedtaksbrev/alternativeMottakere og harAlternativeMottakere i props for å kjøre